### PR TITLE
Update install docs

### PR DESCRIPTION
**Pull request by emerkle826**
**What this PR does**:
  - fix typos in URLs
  - add options for "kubectl apply" to avoid "metadata.annotations: Too long"
  - update "get deploymnet" output example

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [x] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: docs/install/README.md [hunk 1]
```diff
@@ -1028,8 +1028,8 @@ will run on the same Docker network which means that they will have routable IPs
 #### Create kind clusters
 Run `setup-kind-multicluster.sh` as follows:
 
-```sh
-./setup-kind-multicluster.sh --clusters 2 --kind-worker-nodes 4
+```console
+./scripts/setup-kind-multicluster.sh --clusters 2 --kind-worker-nodes 4
 ```
 
 When creating a cluster, kind generates a kubeconfig with the address of the API server
```
Comment (adutra) [inline docs/install/README.md:1035]: I t would be better to wait for cert-manager to be rolled out; after line 1056, add the following:

```console
kubectl rollout status deployment cert-manager-webhook -n cert-manager
```

Comment (emerkle826) [reply docs/install/README.md:1046]: Good call. I've added this to each of the cert-manager install sections.

File: docs/install/README.md [hunk 2]
```diff
@@ -722,8 +722,8 @@ Here is a summary of what the script does:
 Create a `ClientConfig` in the `kind-k8ssandra-0` cluster using the service account 
 token and CA cert from `kind-k8ssandra-1`:
 
-```sh
-./create-clientconfig.sh --namespace k8ssandra-operator --src-kubeconfig build/kubeconfigs/k8ssandra-1.yaml --dest-kubeconfig build/kubeconfigs/k8ssandra-0.yaml --in-cluster-kubeconfig build/kubeconfigs/updated/k8ssandra-1.yaml --output-dir clientconfig
+```console
+./scripts/create-clientconfig.sh --namespace k8ssandra-operator --src-kubeconfig build/kubeconfigs/k8ssandra-1.yaml --dest-kubeconfig build/kubeconfigs/k8ssandra-0.yaml --in-cluster-kubeconfig build/kubeconfigs/updated/k8ssandra-1.yaml --output-dir clientconfig
```
Comment (jsanda) [inline docs/install/README.md:726]: There have been some changes to the `setup-kind-multicluster.sh` script that changed the args we pass to `create-clientconfig.sh`. Take a look at the `create-clientconfig` target in the Makefile for reference.

File: docs/install/README.md [hunk 3]
```diff
@@ -835,7 +846,7 @@ on Docker Hub for a list of available images.
 Install with kubectl:
 
 ```console
-kubectl apply -k github.com/k8ssandra/k8ssandra-operator/config/deployments/control-plane
+kubectl apply --server-side --force-conflicts -k github.com/k8ssandra/k8ssandra-operator/config/deployments/control-plane
```
Comment (jsanda) [inline docs/install/README.md:849]: There is some overlap here with the changes in #437. We need to use `kustomize build | kubectl apply ...`. See #437 for background.

File: docs/install/README.md [hunk 4]
```diff
@@ -1068,6 +1090,17 @@ Install Cert Manager:
 kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.5.3/cert-manager.yaml
```
Comment (jsanda) [inline docs/install/README.md:1090]: cert-manager version needs to be updated.

File: docs/install/README.md [hunk 5]
```diff
@@ -1078,7 +1111,7 @@ kubectx kind-k8ssandra-0
 Now install the operator:
 
 ```console
-kubectl apply -k github.com/k8ssandra/config/deployments/control-plane
+kubectl apply --server-side --force-conflicts -k github.com/k8ssandra/k8ssandra-operator/config/deployments/control-plane
```
Comment (jsanda) [inline docs/install/README.md:1114]: Need to use `kustomize build ... | kubectl apply ...`. See #437 for background.

File: docs/install/README.md [hunk 6]
```diff
@@ -1101,35 +1134,36 @@ Verify that the following CRDs are installed:
...
-```
+```console
 kubectx kind-k8ssandra-1
 ```
 
 Now install the operator:
 
 ```console
-kubectl apply -k github.com/k8ssandra/config/deployments/data-plane
+kubectl apply --server-side --force-conflicts -k github.com/k8ssandra/k8ssandra-operator/config/deployments/data-plane
```
Comment (jsanda) [inline docs/install/README.md:1166]: Need to use `kustomize build ... | kubectl apply ...`. See #437 for background.

File: docs/install/README.md [hunk 7]
```diff
@@ -1183,8 +1217,8 @@ Here is a summary of what the script does:
 Create a `ClientConfig` in the `kind-k8ssandra-0` cluster using the service account 
 token and CA cert from `kind-k8ssandra-1`:
 
-```sh
-./create-clientconfig.sh --namespace k8ssandra-operator --src-kubeconfig build/kubeconfigs/k8ssandra-1.yaml --dest-kubeconfig build/kubeconfigs/k8ssandra-0.yaml --in-cluster-kubeconfig build/kubeconfigs/updated/k8ssandra-1.yaml --output-dir clientconfig
+```console
+./scripts/create-clientconfig.sh --namespace k8ssandra-operator --src-kubeconfig build/kubeconfigs/k8ssandra-1.yaml --dest-kubeconfig build/kubeconfigs/k8ssandra-0.yaml --in-cluster-kubeconfig build/kubeconfigs/updated/k8ssandra-1.yaml --output-dir clientconfig
```
Comment (jsanda) [inline docs/install/README.md:1221]: See the `create-clientconfig` target in the Makefile for how we want to invoke `create-clientconfig.sh`.

**Review by jsanda — CHANGES_REQUESTED** [review]

**Comment by adejanovski** [comment]
@emerkle826, how far are we from getting this merged? Should we shelve this for now?
Also, note that our primary installation documentation is [here](https://github.com/k8ssandra/k8ssandra/tree/docs-v2/docs/content/en/install/local).
Maybe we should just point to https://docs-v2.k8ssandra.io/install/ in order to avoid duplicating information?

**Comment by emerkle826** [comment]
> @emerkle826, how far are we from getting this merged? Should we shelve this for now? Also, note that our primary installation documentation is [here](https://github.com/k8ssandra/k8ssandra/tree/docs-v2/docs/content/en/install/local). Maybe we should just point to https://docs-v2.k8ssandra.io/install/ in order to avoid duplicating information?

This fell off my radar. I'm fine with shelving it. I'll double check through what I have here to make sure the official docs are accurate. Sorry about losing track of this.

**Comment by adejanovski** [comment]
sounds good, no worries

**Pull request closed without merging**